docs/Makefile

```diff
@@ -57,6 +58,7 @@ else
 endif
 
 html:
+        mkdir -p $(STATIC_BUILD_DIR)
```

Comment:
needs a tab instead of spaces, because Makefile

Reply:
oh man, I thought I pushed the changes, I guess it was other repo and not certguard, thanks